src/Form/JsonLdSettingsForm.php

```diff
@@ -50,7 +50,8 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     ];
 
     $rdf_namespaces = '';
-    foreach ($config->get('rdf_namespaces') as $namespace) {
+    $namespaces = $config->get('rdf_namespaces') ?? [];
```

Comment:
This is defined as a non-nullable sequence in the schema: https://github.com/Islandora/jsonld/blob/8db4c553f7babe4c9bdf570d0c6a486dca1a58e1/config/schema/jsonld.schema.yml#L8-L19

Should probably just add an empty array to the default config object?: https://github.com/Islandora/jsonld/blob/3.x/config/install/jsonld.settings.yml

Possibly add a (post-?)update hook to ensure the property is populated (at least with an empty array) in existing installations?

Reply:
The error seems to indicate it was set to null 🤷 

```
Warning: foreach() argument must be of type array|object, null given in Drupal\jsonld\Form\JsonLdSettingsForm->buildForm() (line 53 of modules/contrib/jsonld/src/Form/JsonLdSettingsForm.php).
```

But I can add `[]` to the settings

Reply:
It's not _set_ to null, it's just the `$config->get([...])` returns null for things that are not set.

Reply:
sure but I think this code change will fix the bug reported without any update hooks.

Reply:
... I mean, sure, but do we want to propagate the allowance of invalid config data? Or just fix it once-and-for-all?

Reply:
https://github.com/Islandora/jsonld/pull/85 looks good

Reply:
As in, feel like the idea of implementing a post-update hook is being over estimated in terms of complexity? Something like https://github.com/Islandora/jsonld/pull/85 should do the trick?